oaknational/Oak-Web-Application
# style: force icon sizes into theme spacing values

## Description

- We are heavily leaning towards the practise of limiting style props with union types. This is for two reasons:
  - it improves consistency (eg. rather than having a 32px icon in one place and 34px icon elsewhere)
  - if at some point we move to a static css solution (e.g. vanilla extract or other), we'll need to classify a fixed set of options in order to generate the css
- This PR sets `iconsize` (and a few other props) to be of type `PixelSpacing` (naming can probably be improved), which is the set number of pixels as specified out in figma designs 
- As such, in a bunch of places this PR changes values to fall within the allowed values
- Use `<Card>` and `<CardTitle>` components in BlogList


## How to test

1. Go to {app}
2. Click around
3. You should be happy enough with the sizes of the pixels


## Checklist
